src/radio-button-group.jsx

```diff
@@ -116,6 +116,38 @@ const RadioButtonGroup = React.createClass({
     }
   },
 
+  _radioButtonMapFunc(option) {
+    if (option.props.children === undefined) {
+      let {
+        name,
+        value,
+        label,
+        onCheck,
+        ...other,
+      } = option.props;
+      if (option.type.displayName === 'RadioButton') {
+        return <RadioButton
+          {...other}
+          ref={option.props.value}
+          name={this.props.name}
+          key={option.props.value}
+          value={option.props.value}
+          label={option.props.label}
+          labelPosition={this.props.labelPosition}
+          onCheck={this._onChange}
+          checked={option.props.value === this.state.selected}/>;
+      }
+    }
+
+    return {
+      ...option,
+      props: {
+        ...option.props,
+        children: React.Children.map(option.props.children, this.radioButtonMapFunc, this)
```

Comment:
`this._radioButtonMapFunc`

or rather, rename `_radioButtonMapFunc` to `radioButtonMapFunc`